_source/_posts/2024-08-15-otp-over-sms.md

```diff
@@ -149,6 +149,8 @@ The kind of vendors you need depends on your approach. Below are a few possibili
 
 Here are some vendors you can use to implement the hook in-house or with a consultant.
 
+* [AWS Pinpoint](https://aws.amazon.com/pinpoint/)
```

Comment:
Why was this moved?

Reply:
Folks who talk to customers were reporting that categorizing it in its previous location was causing confusion. It's not clearly incorrect for either section, so I followed the jira's request to move it to the section that works better for readers.